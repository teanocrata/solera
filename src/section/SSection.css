@import url('https://fonts.googleapis.com/css?family=Roboto');

h3 {
  text-transform: uppercase;
  color: forestgreen;
  font-family: "Roboto", sans-serif;
  font-weight: bold;
}

p {
  font-family: "Roboto", sans-serif;
  color: grey;
  font-size: 20px;
}

.img-circle {
  max-width: 60%;
}

@media (max-width: 768px) {
  p {
    font-size: 16px;
  }
}

@media (max-width: 425px) {
  p {
    font-size: 14px;
  }
}
