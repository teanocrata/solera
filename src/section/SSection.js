import React, { Component } from "react";
import { Grid, Row, Col, Image } from "react-bootstrap";
import "./SSection.css";
import ponyGirls from "./pony_girls_square.jpg";
import farmer from "./farmer_square.jpg";
import horseTraining from "./horse_training_square.jpg";

class SSection extends Component {
  render() {
    return (
      <Grid fluid>
        <Row
          className="show-grid"
          style={{
            backgroundColor: "white"
          }}
        >
          <Col md={4} className="slideanim">
            <Image src={ponyGirls} circle />
            <h3>Da de comer a los ponys</h3>
            <p>
              Los caballos y los ponys son animales muy sociables, detectan
              nuestras emociones y son muy curiosos, además ¡les encantan las
              frutisinas!
            </p>
          </Col>
          <Col md={4} className="slideanim">
            <Image src={farmer} circle />
            <h3>Conoce nuestro día a día</h3>
            <p>
              Una escuela de equitación tiene muchas actividades, desde hacer
              las camas, la comida o salir a jugar ¡igual que en casa!
            </p>
          </Col>
          <Col md={4} className="slideanim">
            <Image src={horseTraining} circle />
            <h3>Monta en uno de nuestros ponys</h3>
            <p>
              La actividad estrella de nuestra visita, descubre la sensación de
              conectar con un pony, verás como te conoce, te escucha, y
              disfrutaréis juntos.
            </p>
          </Col>
        </Row>
      </Grid>
    );
  }
}

export default SSection;
