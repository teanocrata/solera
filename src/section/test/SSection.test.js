import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import SSection from "../SSection";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<SSection />, div);
});

test("SSection renders correctly", () => {
  const section = renderer.create(<SSection />).toJSON();
  expect(section).toMatchSnapshot();
});
