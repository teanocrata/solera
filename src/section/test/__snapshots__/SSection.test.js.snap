// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SSection renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="show-grid row"
    style={
      Object {
        "backgroundColor": "white",
      }
    }
  >
    <div
      className="slideanim col-md-4"
    >
      <img
        className="img-circle"
        src="pony_girls_square.jpg"
      />
      <h3>
        Da de comer a los ponys
      </h3>
      <p>
        Los caballos y los ponys son animales muy sociables, detectan nuestras emociones y son muy curiosos, además ¡les encantan las frutisinas!
      </p>
    </div>
    <div
      className="slideanim col-md-4"
    >
      <img
        className="img-circle"
        src="farmer_square.jpg"
      />
      <h3>
        Conoce nuestro día a día
      </h3>
      <p>
        Una escuela de equitación tiene muchas actividades, desde hacer las camas, la comida o salir a jugar ¡igual que en casa!
      </p>
    </div>
    <div
      className="slideanim col-md-4"
    >
      <img
        className="img-circle"
        src="horse_training_square.jpg"
      />
      <h3>
        Monta en uno de nuestros ponys
      </h3>
      <p>
        La actividad estrella de nuestra visita, descubre la sensación de conectar con un pony, verás como te conoce, te escucha, y disfrutaréis juntos.
      </p>
    </div>
  </div>
</div>
`;
